app/gist/controller.js

```diff
@@ -15,6 +15,7 @@ export default Em.Controller.extend({
    */
   buildOutput: Ember.Object.create({ code: '', styles: ''}),
   isBuilding: false,
+  unsaved: true,
```

Comment:
Can we make this a CP on the underlying `model`?  If we can, then the various `setupController` functions implemented here can be removed...